examples/consumer-and-issuer/src/actions/index.ts

```diff
@@ -249,18 +249,20 @@ export const getCredentialCompliantly = async (credentialId: string) => {
 
 export const invokePassportingService = async (payload: unknown) => {
   invariant(process.env.PASSPORTING_SERVICE_URL, "`PASSPORTING_SERVICE_URL` is not set");
-  return await await fetch(process.env.PASSPORTING_SERVICE_URL ?? "", {
+  return await fetch(process.env.PASSPORTING_SERVICE_URL, {
     method: "POST",
     body: JSON.stringify(payload),
     headers: {
       "Content-Type": "application/json",
       Authorization: `Bearer ${process.env.PASSPORTING_SERVICE_API_KEY}`,
     },
   })
-    .then((res) => res.json())
+    .then(async (res) => {
+      return await res.json();
+    })
     .catch((err) => {
       console.error(err);
-      return { ok: false, error: err };
+      return { success: false, error: err };
```

Comment:
```suggestion
      return { success: false, error: {message: err.message, cause: err.cause };
```

And let's also add a return type to make sure we don't accidentally run into these.